tests/console/yast2_apparmor.pm

```diff
@@ -108,6 +108,7 @@ sub run {
             wait_screen_change { send_key 'tab' };
             wait_screen_change { send_key 'end' };    # we need to search for recent toggled element at the of the list
         }
+        wait_still_screen(2);
```

Comment:
I prefer `stilltime => 2` as a lot of people believe this is a timeout because it's in so many other calls

Reply:
Changed as suggested. Tests rerun and updated in this PR.
Thanks for the suggestion/tip!